### ISSUE TITLE
Integrate more optimization algorithms into PEC

### ISSUE THREAD

**Issue by davidt0x** [PrincetonUniversity/PsyNeuLink#2659]
Implement additional support for optimization algorithms from hyperparameter optimization libraries. Potentional options could be optuna, scipy optimize, ray tune, or hyperopt.

**Comment by jdcpni** [comment]
Thanks!  Gather you’ll add one for the advanced methods as well (MCMC, hiearchical and neuro )...

jdc

On May 2, 2023, at 3:23 PM, David Turner ***@***.***> wrote:



Implement additional support for optimization algorithms from hyperparameter optimization libraries. Potentional options could be optuna, scipy optimize, ray tune, or hyperopt.

**Comment by davidt0x** [comment]
Yeah, here is the [project](https://github.com/orgs/PrincetonUniversity/projects/26). I also added a issue label for PEC.
